.gitignore

```diff
@@ -0,0 +1,9 @@
+# Logs
```

Comment:
Mind keeping this out of the commit? I don't include systematic ignores in any of my projects - I prefer to keep things that are not explicitly unique to the project in my global `.gitignore`